Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -522,6 +522,31 @@ class NewTabPageViewController: UIViewController, Themeable {
     ) {
         guard let braveTodaySection = layout.braveTodaySection else { return }
         
+        func _completeLoading() {
```

Comment:
can make it private

Reply:
It is function-scoped, no accessor needed as its not visible outside of the function